portfolio/src/main/webapp/map.html

```diff
@@ -5,13 +5,16 @@
     <link rel="stylesheet" href="common-style.css" />
     <link rel="stylesheet" href="map-style.css" />
     <link href="https://fonts.googleapis.com/css2?family=Raleway&display=swap" rel="stylesheet" />
-    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyARIbGzg1ObeVicxm6Txr1mqhUR4w12LQQ"></script>
+    <script src="config.js"></script>
```

Comment:
I think you should modify the top-level .gitignore file to include the config.js file so that it won't nag you that the file is not committed or let you accidentally commit it. I could be wrong about how this works though.

Reply:
I believe .gitignore already includes config.js (it was also used to build a geochart in a previous pull request).